### PR TITLE
add 2FA step to onboarding

### PR DESCRIPTION
**Pull request by Trott**

### CODE REVIEW

**Review by gireeshpunathil — APPROVED** [review]

File: ONBOARDING.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ invalid and are superseded by the TSC Charter.
 * Add them to the `@pkgjs` org and with an `Owner` role.
 * Add them to the `@pkgjs/node-tsc` team.
 * Invite them to the HackerOne [Node.js team](https://hackerone.com/nodejs/team_members).
+  Have them turn on two-factor authentication after they have joined.
```
Comment (nschonni) [inline ONBOARDING.md:20]: Is this supposed to be a new bullet or a continuation of the HackerOne instructions?

Comment (Trott) [reply ONBOARDING.md:20]: A continuation.

**Review by mcollina — APPROVED** [review]
lgtm

**Review by richardlau — APPROVED** [review]

**Review by targos — APPROVED** [review]

**Review by mhdawson — APPROVED** [review]
LGTM

**Review by mmarchini — APPROVED** [review]

**Review by tniessen — APPROVED** [review]

**Pull request merged**